app/views/simplified-account/settings/service-name/index.njk

```diff
@@ -0,0 +1,63 @@
+{% extends "../../../layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% block pageTitle %}
+  Settings - Service name
+{% endblock %}
+
+{% block side_navigation %}
+  {% include "../_settings-navigation.njk" %}
+{% endblock %}
+
+{% set manage_en_link %}
+  <a class='govuk-link govuk-link--no-visited-state' href='{{ manage_en }}'>Change</a>
+{% endset %}
+
+{% set manage_cy_link %}
+  {% if service_name_cy %}
+    <a class='govuk-link govuk-link--no-visited-state' href='{{ manage_cy }}'>Change</a>
+  {% endif %}
+{% endset %}
+
+{% set add_cy_link %}
+  <a class='govuk-link govuk-link--no-visited-state' href='{{ manage_cy }}'>Add Welsh service name</a>
+{% endset %}
+
+{% block mainContent %}
+  <div class="govuk-grid-column-three-quarters">
+    <h2 class="govuk-heading-l">Service name</h2>
+    {{ govukTable({
```

Comment:
Is there a reason we're not using the [Check Your Answers pattern](https://design-system.service.gov.uk/patterns/check-answers/) for this?

The component used there is `govukSummaryList`.

Reply:
the reason is I did not know there was a component for this already! Will refactor